mcs/mcs/Makefile

```diff
@@ -32,7 +32,7 @@ BUILT_SOURCES = cs-parser.cs
 CLEAN_FILES += y.output
 
 %-parser.cs: %-parser.jay $(topdir)/jay/skeleton.cs
-	$(topdir)/jay/jay $(JAY_FLAGS) < $(topdir)/jay/skeleton.cs $< > jay-tmp.out && mv jay-tmp.out $@
```

Comment:
I don't understand why but ok either way.

Reply:
The old pipe solution produces a 0-byte .cs file on jay failure (this was happening on windows until I fixed the csprojs), which is not noticed by make and results in builds never working again. New solution uses -o and jay will unlink on failure so the file is Missing and make knows to try to rebuild it.